.github/workflows/smoke-tests.yml

```
@@ -0,0 +1,94 @@
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.payload.pull_request.number
+            });
+
+            const conclusion = '${{ job.status }}';
+            const emoji = conclusion === 'success' ? '✅' : '❌';
+            const status = conclusion === 'success' ? 'passed' : 'failed';
+
+            const comment = `${emoji} **Smoke tests ${status}**
+
+Tests ran without API keys - covers ~90% of functionality.
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 82: could not find expected ':'

(syntax-check)